src/library/scala/util/Try.scala

```diff
@@ -273,6 +331,9 @@ final case class Success[+T](value: T) extends Try[T] {
       if (p(value)) this else Failure(new NoSuchElementException("Predicate does not hold for " + value))
     } catch { case NonFatal(e) => Failure(e) }
   override def recover[U >: T](pf: PartialFunction[Throwable, U]): Try[U] = this
+  override def contains[U >: T](value: U): Boolean = this.value == value
+  override def exists(p: T => Boolean): Boolean = p(value)
+  @inline override def forall(p: T => Boolean): Boolean = exists(p)
```

Comment:
Not sure inline means much. If I have a `Success`, would I ever `s.forall(p)` or just `p(s.get)`?